libs/block-editor/src/lib/demo/block-editor.component.scss

```diff
@@ -12,6 +12,11 @@ tiptap-editor {
         display: block;
         height: 100%;
         border-radius: 4px;
-        padding: 1rem 4rem;
+        padding: 1.5rem 4rem;
     }
 }
+
+.highlighted {
```

Comment:
What is this class for?

Reply:
This class is for the highlight of the text to which the link is going to be added.

When you click on the form to add a link, we lost the text selection. That class is added to the text to simulate a text selection.

<img width="382" alt="Screen Shot 2021-11-18 at 2 26 13 PM" src="https://user-images.githubusercontent.com/72418962/142474904-0c6ae2f5-8225-4309-9ee8-3aa3f607289b.png">